src/options/components/navigation/index.jsx

```diff
@@ -1,13 +1,22 @@
 import React, { PropTypes } from 'react'
 import { Link } from 'react-router'
-import styles from './styles'
+import classNames from 'classnames/bind'
+
+import styles from './styles.css'
 
 const Navigation = ({ currentLocation, routes }) => {
     function buildRoutes() {
+        let cx = classNames.bind(styles)
```

Comment:
From the `css-modules` docs about `classNames.bind`:
> Note that in ES2015 environments, it may be better to use the "dynamic class names" approach documented above.

I agree with this, to avoid this extra level of invisible magic.